account_invoice_search_by_reference/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 
 {
     "name": "Account invoice search by reference",
-    "version": "12.0.1.0.0",
+    "version": "13.0.1.0.0",
     "author": "Eficent, Odoo Community Association (OCA)",
     "category": "Invoicing",
     "website": "http://www.eficent.com/",
```

Comment:
https://github.com/OCA/account-invoicing

Reply:
Edited,
Thank you for your review.